ubiquitypress/hyku
# we can know use account settings to to set the value of false to exempt a tenant's record from being indexed to shared-search

https://trello.com/c/EQuv9SXH/605-we-can-know-use-account-settings-to-to-set-the-value-of-false-to-exempt-a-tenants-record-from-being-indexed-to-shared-search

 #####  Post deploy tasks
rake ubiquity_create_parent_account:update['sandbox.repo-test.ubiquity.press']

##### How to test this
1. create a record from sandbox and it should be in both normal and shared search
2.  Do thesame for any  account that has '-demo' in its name or that has the value of false for shared search in account settings when you inspect it in the dashboard.  The work should be in normal search but not shared search.
